lib/modules/datasource/deb/readme.md

```diff
@@ -55,3 +58,50 @@ RUN apt-get update && \
 ```
 
 When the apt package for `gcc` is updated, Renovate updates the environment variable.
+
+```json title="Overwrite deb registryUrl via packageRule"
+{
+  "packageRules": [
+    {
+      "matchDatasources": ["deb"],
+      "matchPackageNames": ["gcc-11"],
+      "registryUrls": [
+        "https://deb.debian.org/debian?suite=stable&components=main,contrib,non-free&binaryArch=amd64"
+      ]
+    }
+  ]
+}
+```
+
+## Artifactory
+
+This datasource can also be used with Artifactory.
+The supported repository types are:
+
+- virtual
+- local
+- remote
+
+If you are using Artifactory, you can use the `deb` datasource with following `registryUrl` format:
+
+```
+https://<host>:<port>/artifactory/<repository-slug>?suite=<suite>&components=<components>&binaryArch=<binaryArch>
```

Comment:
I don't think it doesn't matter if release or suite is used, but it should be consistent with the part below.
```suggestion
https://<host>:<port>/artifactory/<repository-slug>?release=<release>&components=<components>&binaryArch=<binaryArch>
```